src/homepageExperience/components/steps/arduino/ExecuteAggregateQuery.tsx

```diff
@@ -0,0 +1,37 @@
+// Libraries
+import React, {useEffect} from 'react'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+import {keyboardCopyTriggered, userSelection} from 'src/utils/crossPlatform'
+
+const logCopyCodeSnippet = () => {
+  event('firstMile.arduinoWizard.executeAggregateQuery.code.copied')
+}
+
+type OwnProps = {
+  bucket: string
+}
+
+export const ExecuteAggregateQuery = (props: OwnProps) => {
+  const {bucket} = props
+
+  useEffect(() => {
+    const fireKeyboardCopyEvent = event => {
+      if (
+        keyboardCopyTriggered(event) &&
+        userSelection().includes('influx query')
+      ) {
+        logCopyCodeSnippet()
+      }
+    }
+    document.addEventListener('keydown', fireKeyboardCopyEvent)
+    return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
+  }, [])
+
+  return (
+    <>
```

Comment:
I think these fragments can be removed (might apply to all of the Fragments with a single child component)